### PR TITLE
Support the instance segmentation task in projects

### PR DESCRIPTION
**Pull request by zytx121**
This project implements a Mask RCNN for rotated boxes. Benefiting from the BoxType design, we only need to modify the code slightly in mmrotate, and we can support the instance segmentation task. Although the rotated box indicator will drop slightly after adding the mask head, it may help improve the instance segmentation task. 

P0262__1024__824___1414
P0003__1024__123___0

**We hope this project can inspire you and welcome you to explore more uses of mmrotate!**

### CODE REVIEW

File: projects/rotated_mask_rcnn/README.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+```
+
+## Results
+
+<!-- List the results as usually done in other model's README. [Example](https://github.com/open-mmlab/mmrotate/blob/1.x/configs/r3det/README.md#results-and-models)
+You should claim whether this is based on the pre-trained weights, which are converted from the official release; or it's a reproduced result obtained from retraining the model in this project. -->
+
+|         Backbone         |  mAP  | Angle | lr schd | Mem (GB) | Inf Time (fps) | Aug | Batch Size |                                       Configs                                        |         Download         |
+| :----------------------: | :---: | :---: | :-----: | :------: | :------------: | :-: | :--------: | :----------------------------------------------------------------------------------: | :----------------------: |
+| ResNet50 (1024,1024,200) | 72.71 | le90  |   1x    |    -     |       -        |  -  |     2      |  [rotated-mask-rcnn_r50_fpn_1x_dota](confsigs/rotated-mask-rcnn_r50_fpn_1x_dota.py)  | [model](<>) \| [log](<>) |
```
Comment (RangiLyu) [inline projects/rotated_mask_rcnn/README.md:42]: Do you have model weights to upload?

**Comment by fkroeber** [comment]
Thanks for this contribution, I believe it's very valuable to expand the libraries functionality!
One question regarding the obtained accuracies for both models:
Isn't it suspicious that the Oriented Mask-RCNN implementing a dedicated rotated RPN block leads to lower accuracies than the standard rotated Mask-RCNN which uses axis-aligned region porposals in its head?

**Comment by yangxue0827** [comment]
@zytx121

**Comment by zytx121** [comment]
@fkroeber Thank you for sharing, the results are much better than now! May I ask if you are willing to provide us with a PR?

**Comment by tcotte** [comment]
Hello,
I am new to MMRotate. I've found this library while I was searching for an instance segmentation model with oriented bounding boxes. Thank you @zytx121, I think this project will solve my problem. 
I have seen that DOTA dataset type is privileged in MMRotate. But I guess that masks have to be provided. Therefore, I assume that DOTA dataset is not supported by your project. This is why I ask you what type of dataset have to be used in this project ?
